assets/src/dashboard/parts/connected/settings/OffloadMedia.js

```diff
@@ -181,12 +183,12 @@ const OffloadMedia = ({ settings, canSave, setSettings, setCanSave }) => {
 		setCanSave( true );
 
 		if ( offloadEnabled ) {
-			setModal( MODAL_STATE_OFFLOAD );
+			setModal( show_exceed_plan_quota_notice ? MODAL_STATE_EXCEED_PLAN_QUOTA_NOTICE : MODAL_STATE_OFFLOAD );
 
 			return;
 		}
 
-		setModal( MODAL_STATE_ROLLBACK );
+		setModal( show_exceed_plan_quota_notice ? MODAL_STATE_EXCEED_PLAN_QUOTA_NOTICE : MODAL_STATE_ROLLBACK );
```

Comment:
When running the rollback, the modal shouldn't be displayed.

Reply:
I mean we should display the rollback modal, not the warning.

Reply:
Fixed!